scenarios/evaluate-safety/evaluate-protected-material-and-indirect-attack-jailbreak.ipynb

```diff
@@ -0,0 +1,512 @@
+    "Now that we have our dataset, we can evaluate it to see if the indirect attacks resulted in jailbreaks. The `IndirectAttackEvaluator` class can take in the dataset and detects instances of jailbreak. Let's use the `evaluate()` API to run the evaluation and log it to our Azure AI Studio Project."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "indirect_attack_eval = IndirectAttackEvaluator(project_scope=project_scope, credential=DefaultAzureCredential())\n",
+    "file_path = \"indirect_attack_outputs.jsonl\"\n",
+    "result = evaluate(\n",
+    "    data=xpia_file_path,\n",
+    "    evaluators={\n",
+    "        \"indirect_attack\": indirect_attack_eval,\n",
+    "    },\n",
+    "    # Optionally provide your AI Studio project information to track your evaluation results in your Azure AI Studio project\n",
+    "    azure_ai_project=project_scope,\n",
+    "    # Optionally provide an output path to dump a json of metric summary, row level data and metric and studio URL\n",
+    "    output_path=\"./mynewindirectattackevalresults.json\",\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can see that our \"model\" application gives us a defect rate broken down by different behaviors resulting from the jailbreak, showing us that we can't deploy our application just yet. Moving forward, to protect our application against indirect jailbreak attacks, we can add an Azure AI Content Safety Prompt Shield which is a mitigation layer to help annotate and block requests to your model or application that contain known indirect attacks for jailbreak. Let's apply this filter and re-run the simulator and evaluation step to see if it helps with our defect rate."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "filtered_indirect_attack_outputs = await indirect_attack_simulator(\n",
+    "    target=xpia_callback,  # now with the Prompt Shield attached to our model deployment\n",
+    "    scenario=AdversarialScenario.ADVERSARIAL_INDIRECT_JAILBREAK,\n",
+    "    max_simulation_results=10,\n",
+    "    max_conversation_turns=3,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "filtered_indirect_attack_result = evaluate(\n",
+    "    data=filtered_indirect_attack_outputs,\n",
+    "    evaluators={\"indirect_attack\": indirect_attack_eval},\n",
+    "    # Optionally provide your AI Studio project information to track your evaluation results in your Azure AI Studio project\n",
+    "    azure_ai_project=project_scope,\n",
+    "    # Optionally provide an output path to dump a json of metric summary, row level data and metric and studio URL\n",
+    "    output_path=\"./myindirectattackevalresults.json\",\n",
+    ")"
+   ]
+  }
+ ],
```

Comment:
Can you please add one last cell markdown to conclude the sample:
"In conclusion, we've walked through how to generate test datasets using the simulation framework and our safety evaluation framework. See our documentation for more details and additional functionality on [simulation ](https://aka.ms/advsimulatorhowto) and [evaluation](https://aka.ms/azureaistudiosafetyevalhowto)."